src/components/app.tsx

```diff
@@ -339,5 +348,5 @@ export class App extends React.PureComponent<IProps, IState> {
   private handleLoadPlugins = () => {
     this.setState({ pluginsLoaded: true });
   }
-
+  
 }
```

Comment:
I think this module was saved without trailing spaces removed.  I would remove trailing spacing using the settings in VS Code.
https://stackoverflow.com/questions/30884131/remove-trailing-spaces-automatically-or-with-a-shortcut